phys/module_radiation_driver.F

```diff
@@ -1609,7 +1618,29 @@ SUBROUTINE radiation_driver (                                  &
      ENDIF
 #endif
 
-        
+     IF ( cu_physics == BMJSCHEME ) THEN
```

Comment:
@rmfonsecaweb Do you also need to check if bmj_rad_feedback is on?

Reply:
We do not believe this is necessary as when "bmj_rad_feedback" is .FALSE. the arrays QCCONV, QICONV and CCLDFRA are zero and hence QC, QI and CLDFRA are not modified. However, following the reviewer's comment, we will modify the file checking for the referred flag. Thank you!